test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -107,18 +107,20 @@ type testConfig struct {
 	numNodes            int
 	mutatedNodeTemplate *v1.Node
 	mutatedPodTemplate  *v1.Pod
-	schedulerSupport    *factory.ConfigFactoryArgs
+	schedulerConfig     *factory.Config
+	clientset           clientset.Interface
 	destroyFunc         func()
 }
 
 // getBaseConfig returns baseConfig after initializing number of nodes and pods.
 func getBaseConfig(nodes int, pods int) *testConfig {
-	schedulerConfigArgs, destroyFunc := mustSetupScheduler()
+	schedulerConfig, destroyFunc, clientset := mustSetupScheduler()
```

Comment:
can we have a separate helper for creating a clientset? It looks odd that `mustSetupCluster` returns a clientset. It reads better to have:

```
clientset := testClientset()
destroyFunc := mustSetupScheduler(clientset)
```

Reply:
I don't think it is necessary, and we always have to call these function at the time. I'll wait for other folks opinion on this.